rush-plugins/rush-buildxl-graph-plugin/src/test/GraphProcessor.test.ts

```diff
@@ -48,7 +39,9 @@ describe(GraphProcessor.name, () => {
   });
 
   it('should process debug-graph.json into graph.json', () => {
-    let prunedGraph: IGraphNode[] = graphParser.processOperations(new Set<Operation>(getDebugOperationMap()));
+    let prunedGraph: IGraphNode[] = graphParser.processOperations(
+      new Set<Operation>(debugGraph.OperationMap as unknown as Operation[])
```

Comment:
Why the cast?

Reply:
The test data is a JSON object, missing a bunch of thigns.